src/modules/saveReminder/SaveReminder.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    Observable as KnockoutObservable,
+} from 'knockout';
+import { Saveable } from '../DataStore/common/Saveable';
+import * as GameConstants from '../GameConstants';
+import Notifier from '../notifications/Notifier';
+import Settings from '../settings';
+
+export default class SaveReminder implements Saveable {
+    public static counter = 0;
+
+    saveKey = 'saveReminder';
+
+    defaults: Record<string, any> = {};
+
+    public lastDownloaded: KnockoutObservable<number> = ko.observable(0).extend({ numeric: 0 });
+    public lastReminder: KnockoutObservable<number> = ko.observable(0).extend({ numeric: 0 });
+
+    // eslint-disable-next-line @typescript-eslint/no-useless-constructor, @typescript-eslint/no-empty-function
+    constructor() {}
+
+    static tick() {
+        this.counter = 0;
+
+        // Check if reminders are disabled
+        if (!+Settings.getSetting('saveReminder').value) return;
+
+        let timeSinceSaveReminder = App.game.statistics.secondsPlayed() - Math.max(App.game.saveReminder.lastDownloaded(), App.game.saveReminder.lastReminder());
+        // Adjust to measure against ms
+        timeSinceSaveReminder *= 1000;
+
+        if (timeSinceSaveReminder >= +Settings.getSetting('saveReminder').value) {
+            // Show reminder notification
+            Notifier.notify({
+                title: 'Save Reminder',
+                message: `It has been ${GameConstants.formatTimeShortWords(+Settings.getSetting('saveReminder').value)} since your last save download, Would you like to download a backup now?
```

Comment:
Should we add a "This interval can be changed in the settings"? It seams like something a lot of people want to customize, but i don't think it's super intuitive that it's something you can change, so we might get some questions about it.
But i am really not sure, if i think this is a good idea or not...

Reply:
I think it would just add too much clutter, and don't think every bit of information needs to be spoon fed to people, because even if we did, we would still get people coming to ask about blue coin.